Userland/Applets/ClipboardHistory/ClipboardHistoryModel.cpp

```diff
@@ -91,21 +96,23 @@ GUI::Variant ClipboardHistoryModel::data(const GUI::ModelIndex& index, GUI::Mode
         return data_and_type.mime_type;
     case Column::Size:
         return AK::human_readable_size(data_and_type.data.size());
+    case Column::Time:
+        return time.to_string("%Y-%m-%d %H:%M:%S");
```

Comment:
Minor, but this could just be `time.to_string();` as this is the default time format string

Reply:
Indeed!